packages/docs-page/content-api/index.ts

```diff
@@ -1,27 +1,28 @@
 const MKTG_CONTENT_API = process.env.MKTG_CONTENT_API
-const MKTG_CONTENT_API_TOKEN = process.env.MKTG_CONTENT_API_TOKEN
-
-const DEFAULT_HEADERS = {
-  headers: {
-    Authorization: `Bearer ${MKTG_CONTENT_API_TOKEN}`,
-  },
-}
 
 // Courtesy helper for warning about missing env vars during development
 const checkEnvVarsInDev = () => {
   if (process.env.NODE_ENV === 'development') {
-    if (!MKTG_CONTENT_API || !MKTG_CONTENT_API_TOKEN) {
+    if (!MKTG_CONTENT_API) {
       const message = [
         'Missing environment variables required to fetch remote content:',
-        !MKTG_CONTENT_API ? '  - `MKTG_CONTENT_API`' : false, 
-        !MKTG_CONTENT_API_TOKEN ? '  - `MKTG_CONTENT_API_TOKEN`' : false,
-        'Reach out to #team-web-platform to get the proper values.'
-      ].filter(Boolean).join('\n')
+        !MKTG_CONTENT_API ? '  - `MKTG_CONTENT_API`' : false,
+        'Reach out to #team-web-platform to get the proper values.',
+      ]
+        .filter(Boolean)
+        .join('\n')
```

Comment:
nit: Since this is only one env variable now, we could simplify the message construction.